test/access_control.sh

```diff
@@ -51,19 +65,30 @@ function cleanup() {
         check_es_acls
         oc -n $LOGGING_NS logs -c elasticsearch $espod > $ARTIFACT_DIR/es.log
         oc -n $LOGGING_NS exec -c elasticsearch $espod -- logs >> $ARTIFACT_DIR/es.log
-        curl_es_pod $espod /project.access-control-* -XDELETE > /dev/null
+        curl_es_pod $espod /logs-app* -XDELETE > /dev/null
     fi
     for proj in access-control-1 access-control-2 access-control-3 ; do
         oc delete project $proj 2>&1 | artifact_out
         os::cmd::try_until_failure "oc get project $proj" 2>&1 | artifact_out
     done
     # this will call declare_test_end, suite_end, etc.
     os::test::junit::reconcile_output
-    exit $result_code
 }
 
 trap cleanup EXIT
 
+if [ "${DO_SETUP:-true}" == "true" ] ; then
+	os::log::info "Deploying cluster logging..."
+	${repo_dir}/hack/deploy-logging.sh
+fi
+
+espod=$( get_es_pod es )
+esopspod=$( get_es_pod es-ops )
+esopspod=${esopspod:-$espod}
```

Comment:
Is `es-ops` still used in ES6?

Reply:
No.   anything "ops" related can be removed

Reply:
+1 we can remove this -- we haven't used it in 4.x at all

Reply:
if removing "ops" pod here, can we remove checks which use ops pod also? 
e.g. `test_message="$user has improper access to .operations.* indices from es"` etc